src/ucp/core/ucp_worker.c

```diff
@@ -1972,6 +1972,17 @@ static void ucp_worker_destroy_configs(ucp_worker_h worker)
     worker->rkey_config_count = 0;
 }
 
+static void
+ucp_worker_vfs_show_debug_address(void *obj, ucs_string_buffer_t *strb)
+
+{
+    ucp_worker_h worker = obj;
+
+    UCS_ASYNC_BLOCK(&worker->async);
+    ucs_string_buffer_appendf(strb, "%p\n", worker);
```

Comment:
lock is not needed, since we don't access worker fields
can reuse ucp_context_vfs_show_debug_address